src/main/java/com/gl/donate_receive/donate_receive/dto/UserCreateDto.java

```diff
@@ -0,0 +1,18 @@
+package com.gl.donate_receive.donate_receive.dto;
+
+import com.gl.donate_receive.donate_receive.model.Role;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Data
+public class UserCreateDto {
+	@NotNull
+	private String login;
```

Comment:
login should be 320 and password should be limited to 128
![image](https://user-images.githubusercontent.com/108729712/177537413-3b21ef98-14f4-483c-8d99-c37989f1a11e.png)